Meta/Azure/Caches.yml

```diff
@@ -18,13 +18,13 @@ steps:
     - ${{ if eq(parameters.toolchain, 'clang') }}:
       - task: Cache@2
         inputs:
-          key: '"toolchain" | "x86_64" | Toolchain/BuildClang.sh | Toolchain/Patches/*[!gcc].patch | Userland/Libraries/LibC/**/*.h | Userland/Libraries/LibPthread/**/*.h'
+          key: '"toolchain" | "x86_64" | Toolchain/BuildClang.sh | Toolchain/Patches/llvm*.patch | Toolchain/CMake/*.cmake | Userland/Libraries/LibC/**/*.h | Userland/Libraries/LibPthread/**/*.h'
```

Comment:
This also makes the clang Toolchain hash independent of the gdb patch. Irrelevant for CI, though it raises two questions

whether we should build gdb separately from the gcc Toolchain

whether we should exclude that patch from the gcc Toolchain cache as well